docs/getting-started/tutorials/auth-architecture/enterprise-architecture.md

```diff
@@ -0,0 +1,15 @@
+---
+id: enterprise-architecture 
+title: Enterprise Architecture with edge gateways
+sidebar_label: Enterprise Architecture
+---
+
+## Definition
+
+Entrprise Architecture with edge gateways
+- Use edge gateway(s) to centralize user resolution, API Keys, Ingresses (differentiate Ingresses exposed on the internet vs internal)
+- What if I have multiple IDPs but still want to centralize -> exploit edge gateways
```

Comment:
the term "exploit", especially in computer science, is often used with a negative accent (in the sense of abuse). I'd suggest using other terms, such as "leverage" which is generally what you would you use this kind of sentence